specification/storage/resource-manager/Microsoft.Storage/stable/2025-01-01/blob.json

```diff
@@ -0,0 +1,1580 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2025-01-01",
```

Comment:
Please also update the readme.me to include the new API version.
Please refer this PR https://github.com/Azure/azure-rest-api-specs/pull/31466